README.md

```diff
@@ -329,6 +349,13 @@ Then to install the required tools (or update, following a change to
 
 ```shell
 mise install
+mise exec python -- pip install pre-commit
+```
+
+Next ensure you have the pre-commit hook installed by running
```

Comment:
I think we should add something here about this step being optional, I don't think we should imply that it's required to install the hooks to set up the project.